### ISSUE TITLE
Reopen an article to the last page read

### ISSUE THREAD

**Issue by Ygster** [pmartin/plop-reader#66]
When I reopen an article which I had already read a part, I am on the first page not on the page I was when I stop my reading.
Is it possible to remember the page as the ebook reader do with the books ?

**Comment by pmartin** [comment]
I've given it a (long) shot, but didn't manage anything :-(

I was hoping `OpenBook()` would do this by itself, especially as it creates *navigation history* files like it does for other *normal* books (see in `/mnt/ext1/system/state/system/share/plop-reader/entries-epub/`), but it seems those files are either not re-used when re-opening the reader, or don't contain the actual position.

*Maybe* the second parameter to `OpenBook()` (or the second and third parameters to `OpenBook2()`) would allow one to specify a position, but I have no idea whan *parameters* are available / recognized by the reader application :-|

Running `ps -wl` it seems the `/ebrmain/bin/AdobeViewer.app` (the actual reader, I suppose) is called the exact same way than for *normal* books: it only receives one parameter, the full path to the epub file.

So, basically, at least for now, I have no idea why this doesn't work :-(
I'll keep this issue in my *just in case*, but cannot make any promise...
